### DISCUSSION TITLE
Fail to access to port 8080 remotely, but ok from localhost

### DISCUSSION THREAD

**Discussion by robbynie** [nocodb/nocodb#1526] [Q&A]
I deployed docker nocodb on some public cloud. and  it looks like to access Ok from local:

```
[root@ecs-274467 ~]# curl -k http://localhost:8080
Found. Redirecting to /dashboard
```
But failed to access from internet, How can I configure the web to be available for remote access?
Thanks.

**Comment by o1lab** [comment]
You would have to look up cloud provider docs on how to open that port to outside access.

**Reply by robbynie** [reply]
> You would have to look up cloud provider docs on how to open that port to outside access.

A public IP address was given by cloud provider, I was able to config other service to be accessed outside, like Nginx etc...
That is why I guess some configuration inside nocodb...

Do you mean by default, the nocodb docker can be access by default?

I setup this followed the official installation guide.

`docker run -d --name nocodb -p 8080:8080 nocodb/nocodb:latest`
